wiki/Help_centre/Account_restrictions/vi.md

```diff
@@ -23,22 +17,22 @@ Những chức năng sau sẽ bị tắt đối với tài khoản bị hạn ch
 
 Nếu tài khoản bạn bị hạn chế, đừng hoảng hốt. Hãy làm theo các bước sau:
 
-- **Chờ 24 tiếng.** Đôi khi bạn bị hạn chế bởi quy trình tự động nào đó. Chúng tôi luôn kiểm tra các ca hạn chế mới mỗi ngày, xin vui lòng chờ 24 tiếng, nếu bạn bị hạn chế nhầm, thì bạn có thể sẽ được gỡ trong khoảng thời gian đó.
+- **Chờ 24 tiếng.** Đôi khi bạn bị hạn chế bởi quy trình tự động nào đó. Chúng tôi luôn kiểm tra các ca hạn chế mới mỗi ngày, xin vui lòng chờ 24 tiếng - nếu bạn bị hạn chế nhầm, thì bạn có thể sẽ được gỡ trong khoảng thời gian đó.
 - **Liên hệ  [đội ngũ hỗ trợ tài khoản](/wiki/People/The_Team/Account_support_team) của chúng tôi.** Nếu tài khoản của bạn vẫn bị hạn chế sau khoảng thời gian trên, vui lòng gửi email tới [accounts@ppy.sh](mailto:accounts@ppy.sh) **từ email liên kết với tài khoản** để hỏi thêm chi tiết về việc bị hạn chế. Nhớ ghi kèm tên tài khoản, cùng với bất cứ thông tin gì bạn có thể cung cấp thêm như là lý do bạn nghĩ mình bị hạn chế. Bạn cung cấp càng nhiều thông tin, chúng tôi càng dễ giải quyết vấn đề cho bạn.
 - Nếu bạn chắc chắn rằng mình bị hạn chế oan, vui lòng liên hệ cho chúng tôi ngay lập tức để chúng tôi có thể xử lý vụ việc của bạn.
 
-Nếu tài khoản của bạn bị hạn chế bởi vì vi phạm [nội quy](/wiki/Rules), bạn bị bắt buộc phải *chờ ba tháng* trước khi được phép xin khiếu nại.
+Nếu tài khoản của bạn bị hạn chế bởi vì vi phạm [nội quy](/wiki/Rules), bạn phải chờ tối thiểu **ba tháng** trước khi có thể xin khiếu nại.
 
-Nếu tài khoản gốc của bạn chưa bị hạn chế, bạn có thể đăng nhập lại vào đó và chơi game như bình thường, nhưng hãy đảm bảo rằng bạn sẽ không đăng nhập vào các tài khoản phụ từ nay về sau, vì làm vậy sẽ chỉ gia tăng thêm hình phạt của bạn thôi.
+Nếu tài khoản gốc của bạn chưa bị hạn chế, bạn có thể đăng nhập vào đó và tiếp tục chơi như bình thường, nhưng hãy đảm bảo rằng bạn sẽ không đăng nhập vào các tài khoản khác từ nay về sau, vì làm vậy sẽ chỉ gia tăng thêm hình phạt của bạn.
 
-Nếu khiếu nại xin gỡ hạn chế của bạn được chấp nhận và đang chơi tự nhiên bị hạn chế lại, đừng hoảng hốt! Hệ thống tự động có thể vô tình đánh dấu bạn, nhưng bạn sẽ được gỡ hạn chế trong 24 tiếng nếu đó thực sự là lý do. Hãy thoải mái gửi email hoặc tweet để chúng tôi để ý và gỡ hạn chế nhanh hơn.
+Nếu khiếu nại xin gỡ hạn chế của bạn được chấp nhận và đang chơi tự nhiên bị hạn chế lại, đừng hoảng hốt! Hệ thống tự động có thể vô tình đánh dấu bạn, nhưng bạn sẽ được gỡ hạn chế trong 24 tiếng nếu đó thực sự là lý do. Hãy thoải mái gửi email hoặc tweet để chúng tôi để ý và được gỡ hạn chế nhanh hơn.
```

Comment:
```suggestion
Nếu khiếu nại xin gỡ hạn chế của bạn được chấp nhận và đang chơi tự nhiên bị hạn chế lại, đừng hoảng hốt! Hệ thống tự động có thể vô tình đánh dấu bạn, nhưng bạn sẽ được gỡ hạn chế trong 24 tiếng nếu đó thực sự là lý do. Hãy thoải mái gửi email hoặc tweet để chúng tôi để ý và gỡ hạn chế nhanh hơn.
```

Should it be like this? I think the original is fine - most of the time, I'd rather associate the `gỡ hạn chế nhanh hơn` part with `chúng tôi`, since it is nearer.